integration-tests/api/__tests__/store/product-category.ts

```diff
@@ -232,6 +232,47 @@ describe("/store/product-categories", () => {
       )
     })
 
+    it("gets list of product category with all childrens when include_descendants_tree=true", async () => {
+      const api = useApi()
+
+      const response = await api.get(
+        `/store/product-categories?parent_category_id=null&include_descendants_tree=true`,
+      )
+
+      expect(response.status).toEqual(200)
+      expect(response.data.count).toEqual(1)
+      expect(response.data.product_categories).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            id: productCategoryParent.id,
+            parent_category: null,
+            rank: 0,
+            category_children: [
+              expect.objectContaining({
+                id: productCategory.id,
+                parent_category_id: productCategoryParent.id,
+                rank: 0,
+                category_children: [
+                  expect.objectContaining({
+                    id: productCategoryChild4.id,
+                    parent_category_id: productCategory.id,
+                    category_children: [],
+                    rank: 2
```

Comment:
question: Since it should return them all, don't we have a rank 0 and 1 here?

Reply:
for store, we should only return active and non internal categories. 0 and 1 shouldn't be here.

Reply:
Got it, didn't see there were not active ..